docs_src/data_block.ipynb

```diff
@@ -1023,6 +1023,356 @@
     "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```

Comment:
<p>Same, don't document `__len__` as it's standard python. </p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
I got it.